src/preferences/colorpaletteeditormodel.cpp

```diff
@@ -102,9 +102,9 @@ void ColorPaletteEditorModel::setColorPalette(const ColorPalette& palette) {
 
     // Make a map of hotcue indices
     QMap<int, int> hotcueColorIndicesMap;
-    QList<unsigned int> hotcueColorIndices = palette.getHotcueIndices();
-    for (int i = 0; i < hotcueColorIndices.size(); i++) {
-        int colorIndex = hotcueColorIndices.at(i);
+    QList<int> colorIndicesByHotcue = palette.getIndicesByHotcue();
+    for (int i = 0; i < colorIndicesByHotcue.size(); i++) {
+        int colorIndex = colorIndicesByHotcue.at(i);
         hotcueColorIndicesMap.insert(colorIndex, i);
     }
 
```

Comment:
Add documentation to the header file:
"Model for the QTableView in ColorPaletteEditor"

Reply:
You forgot this one.

Reply:
You have commented here the cpp file. In the header file we hae already comments: 
```
// Model that is used by the QTableView of the ColorPaletteEditor.
// Takes of displaying palette colors and provides a getter/setter for
// ColorPalette instances. 
```

Reply:
Sorry, I didn't see that in the diff for this PR so I missed it. But yeah there's nothing you need to do for this.